app/controllers/users.js

```diff
@@ -1,26 +1,52 @@
 const util = require('util');
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
 
-const User = require('../models').users;
 const logger = require('../logger');
 const error = require('../errors');
-const { hashPassword } = require('../services/users');
+const { hashPassword, findByEmail, createUser } = require('../services/users');
 
-const { userCreationError } = error;
+const { userCreationError, userLoginError } = error;
 
-exports.postUser = (req, res, next) => {
+exports.signUpUser = (req, res, next) => {
   const { first_name: firstName, last_name: lastName, email, password } = req.body;
```

Comment:
For these things, we use `mappers`. The mappers transform the request data to our data to create a more declarative code. For example, in this case, you can create a folder `mappers`, create a file `users` and create a method `signUpMapper` which changes the case (snake_case to camelCase).
Change this in the next PR, please! You can do the same thing for the user and password in signIn.